llvm/lib/Transforms/Coroutines/CoroFrame.cpp

```diff
@@ -688,9 +688,9 @@ static DIType *solveDIType(DIBuilder &Builder, Type *Ty,
 static void buildFrameDebugInfo(Function &F, coro::Shape &Shape,
                                 FrameDataInfo &FrameData) {
   DISubprogram *DIS = F.getSubprogram();
-  // If there is no DISubprogram for F, it implies the Function are not compiled
-  // with debug info. So we also don't need to generate debug info for the frame
-  // neither.
+  // If there is no DISubprogram for F, it implies the function is compiled
+  // without debug info. So we also don't generate debug info for the frame,
+  // either.
```

Comment:
```suggestion
  // without debug info. So we also don't generate debug info for the frame.
```
The "also don't" and "either" are redundant.
Arguably the comment rewording belongs in its own patch, especially as there are no other changes to this file, but I'm fine with including it, especially as it matches another comment elsewhere.